worlds/sc2/MissionTables.py

```diff
@@ -734,4 +760,4 @@ def get_campaign_potential_goal_missions(campaign: SC2Campaign) -> List[SC2Missi
 
 
 def get_no_build_missions() -> List[SC2Mission]:
-    return [mission for mission in SC2Mission if not mission.build]
+    return [mission for mission in SC2Mission if mission.flags & MissionFlags.NoBuild]
```

Comment:
instead of using `mission.flag & flag` I'd prefer having a method (`has_flag()`)

Reply:
I mean, using `in` or `&` is the standard way to deal with sets and enum flags / int flags across Python. It's also used elsewhere in Archipelago. `&` is also the standard way to handle enum flags in other languages, like C.

I don't see the point of adding an additional syntax when `in` and `&` already cover all simple use-cases, allow more expressivity when dealing with multiple flags (e.g. can check if `(MissionFlags.NoBuild|MissionFlags.Kerrigan in mission.flags)` in one call), and are just generally the standard.

Reply:
Switched this to use `in` so it looks more like working with other collections. For single-flags, there's no difference between `in` and `&`. I can definitely see myself using `&` for future masks / mission group definitions, though.